src/components/table/table-column-resize/resizable-table-column.component.less

```diff
@@ -17,6 +21,10 @@
 
     .ux-resizable-table-column {
         position: relative;
+
+        .ux-resizable-table-column-content {
+            text-align: left;
```

Comment:
@donna-marijne need to add text-align left so rtlcss flips, in general all the table columns i had to do this for. bootstrap has a th text-align left by default and i have no way to RTL CSS bootstrap but i'm pretty sure that already exists out there in the wild.